connector/client/client.go

```diff
@@ -84,18 +83,30 @@ func (client *ConnectorClient) Post(target url.URL, activity schema.Activity) er
 
 	replyClient := &http.Client{}
 
-	resp, err := replyClient.Do(req)
-	if err != nil || resp.StatusCode != http.StatusCreated {
+	return client.checkRespError(replyClient.Do(req))
+}
+
+func (client *ConnectorClient) checkRespError(resp *http.Response, err error) error {
+	allowedResp := []int{http.StatusOK, http.StatusCreated, http.StatusAccepted}
+	if err != nil {
 		return customerror.HTTPError{
-			StatusCode: resp.StatusCode,
-			HtErr:      err,
-			Body:       resp.Body,
+			HtErr: err,
```

Comment:
Response object would be empty in this case ? Should we embed response attributes here as well as do below ?

Reply:
response would be nil in this case